content/node/sequelize/_index.md

```diff
@@ -93,44 +92,60 @@ In the database server logs, the comment's fields are:
 * URL-quoted except for the equals(`=`) sign e.g `route='%5Epolls/%24'`. so should be URL-unquoted
 
-Field|Format|Description|Example
----|---|---|---
-`client_timezone`|`<string>`|URL quoted name of the timezone used when converting a date from the database into a JavaScript date|`'+00:00'`
-`db_driver`|`<sequelize>`|URL quoted name and version of the database driver|`db_driver='sequelize'`
-`route`|`<the route used>`|The URL-quoted route used to match the express.js controller|`route='%5E%2Fpolls%2F`
+| Field             | Format                 | Description                                                                                          | Example                                                                 |
+|-------------------|------------------------|------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------|
+| `client_timezone` | `<string>`             | URL quoted name of the timezone used when converting a date from the database into a JavaScript date | `'+00:00'`                                                              |
+| `db_driver`       | `<sequelize>`          | URL quoted name and version of the database driver                                                   | `db_driver='sequelize'`                                                 |
+| `route`           | `<the route used>`     | URL quoted route used to match the express.js controller                                             | `route='%5E%2Fpolls%2F`                                                 |
+| `traceparent`     | `<traceparent header>` | URL quoted [W3C `traceparent` header](https://www.w3.org/TR/trace-context/#traceparent-header)       | `traceparent='00-3e2914ebce6af09508dd1ff1128493a8-81d09ab4d8cde7cf-01'` |
+| `tracestate`      | `<tracestate header>`  | URL quoted [W3C `tracestate` header](https://www.w3.org/TR/trace-context/#tracestate-header)         | `tracestate='rojo%253D00f067aa0ba902b7%2Ccongo%253Dt61rcWkgMzE'`        |
 
 #### Options
-When creating the middleware, one can optionally toggle attributes to be set in the comments by passing in the option `include` which is a map
+When creating the middleware, one can optionally configure the injected
+comments by passing in the `include` and `options` objects:
 
 ```javascript
-wrapMainSequelizeAsMiddleware(Sequelize, include={...});
+wrapMainSequelizeAsMiddleware(Sequelize, include={...}, options={...});
 ```
 
-Field|On by default
----|---
-client_timezone|No
-db_driver|No
-route|Yes
-tracestate|Yes
-traceparent|Yes
+##### `include`
+A map of values to be optionally included in the SQL comments.
+
+| Field           | On by default |
+| --------------- | ------------- |
+| client_timezone | No            |
+| db_driver       | No            |
+| route           | Yes           |
+| traceparent     | No            |
+| tracestate      | No            |
```

Comment:
As much as we can, please use the tick or X as I did before (I understand this change had just Yes or No) but it is very important to be able to quickly skim over :)